### PR TITLE
created mentorship link on on boarding page

### PR DESCRIPTION
**Pull request by thandiumuzi**

### CODE REVIEW

File: content/department-processes/team-onboarding/_index.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Channels to join:
  - #meetups-and-events - info about meetups etc that are happening
  - relevant cohort channels (e.g. c14-data, c13_data)
 
+#### Mentorship 
+Here's what you have to know about the role of a Mentor,what is expected from both you and your Mentees.
+Here is the presentation : https://umuzi-org.github.io/tech-department/workshops/how-to-be-a-great-mentor/
```
Comment (ry-oc) [inline content/department-processes/team-onboarding/_index.md]: Would be better if you used a content link as opposed to a static link

Comment (thandiumuzi) [reply content/department-processes/team-onboarding/_index.md]: changes made

**Comment by ry-oc** [comment]
@thandiumuzi something weird happened on this PR (Look at files changes), the best thing to do is to close it

**Comment by ry-oc** [comment]
@thandiumuzi something went wrong here if you look at the files changed you will see it went mad. We should close this PR and redo it. We can go over it on Monday.

**Comment by ry-oc** [comment]
Closed to to git weirdness

**Pull request closed without merging**